cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java

```diff
@@ -333,7 +333,7 @@ public void initialize(NGSIEvent event) throws CygnusBadConfiguration {
             entityType = event.getEntityTypeForNaming(enableGrouping, enableNameMappings);
```

Comment:
An entry in CHANGEX_NEXT_RELEASE regarding the changes in this PR should be provided.

Reply:
Fixed in [2605b9a](https://github.com/telefonicaid/fiware-cygnus/pull/1816/commits/2605b9a29dd54b7cdfb9188399326aa7d63db46f)